release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -490,6 +490,70 @@ For more information, see xref:../registry/configuring_registry_storage/configur
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-google-cloud-file.adoc[Google Compute Platform Filestore CSI Driver Operator].
+
+[id="ocp-4-17-storage-azure-file-snapshot"]
+==== Azure File CSI supports snapshots (Technology Preview)
+{product-title} 4.17 introduces volume snapshot support for the Microsoft Azure File Container Storage Interface (CSI) Driver Operator. This capability is supported as a Technology Preview feature.
+
+For more information, see xref:../storage/container_storage_interface/persistent-storage-csi.adoc#csi-drivers-supported_persistent-storage-csi[CSI drivers supported by {product-title}] and xref:../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc[CSI volume snapshots].
+
+[id="ocp-4-17-storage-multi-vcenter-support"]
+==== Multiple vCenter support for vSphere CSI (Technology Preview)
+{product-title} v4.17 introduces the ability to deploy {product-title} across multiple vSphere clusters (vCenters). This feature is supported with Technology Preview status.
+
+Multiple vCenters can only be configured during installation. The maximum number of supported vCenter clusters is three.
+
+// For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-multi-vcenter-support-overview_persistent-storage-csi-vsphere[Multiple vCenter support for vSphere CSI] and xref:../installing/installing_vsphere/installation-config-parameters-vsphere.adoc[Installation configuration parameters for vSphere].
+
+[id="ocp-4-17-storage-selinuxmount-dev-preview"]
+==== RWX/RWO SELinux Mount (Developer Preview)
+Pods might take a very long time to start when the volume contains a large number of files. To avoid SELinux labeling issues while keeping SELinux confining, you can enable the ReadWriteMany/ReadWriteOnce (RWX/RWO) SELinux Mount feature. Be advised that the RWX/RWO SELinux Mount feature is a Developer Preview feature. It is not supported by Red Hat, and you should not enable this feature set on production or clusters that you plan to maintain over time.
+
+:FeatureName: RWX/RWO SELinux Mount
+include::snippets/technology-preview.adoc[]
+
+For more information about the RWX/RWO SELinux Mount feature, including how to enable it, see link:https://access.redhat.com/articles/7087028[RWX/RWO SELinux Mount feature Knowledge Centered Service article].
+
+[id="ocp-4-17-storage-migrating-cns-vols-between-datastores-dev-preview"]
+==== Migrating CNS volumes between datastores with CLI (Developer Preview)
+In {product-title} 4.17, if you are running out of space in your current datastore, or want to move to a more performant datastore, you can migrate volumes between datastores. Be advised that this feature is a Developer Preview feature. It is not supported by Red Hat.
```

Comment:
mention the tool name "cns-migration" so that it's perfectly clear what tool we are referring to